db/seeds/coursemology_dev.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'factory_girl_rails'
+require 'coursemology/polyglot'
+
+ActsAsTenant.with_tenant(Instance.default) do
+  # Set the stamper for creator and updater fields
+  admin = User::Email.find_by_email('test@example.org').user
+  User.stamper = admin
+
+  # Create courses
+  open_course = FactoryGirl.create(:course, :opened)
```

Comment:
Sorry, have a few more suggestions:
1. Let `admin` be owner of this course
2. Let `admin` be `course_staff` of another course (maybe `published_course`)
3. Add some indication to the course title (similar to the assessment titles)